src/main/kotlin/com/openlattice/assembler/AssemblerConnectionManager.kt

```diff
@@ -555,6 +565,18 @@ class AssemblerConnectionManager(
         }
     }
 
+    private fun revokeConnectAndSchemaUsage(datasource: HikariDataSource, dbname: String, userId: String) {
+        val dbUser = DataTables.quote(userId)
+        logger.info("Removing user {} from database {} and schema usage of $MATERIALIZED_VIEWS_SCHEMA", userId, dbname)
+
+        datasource.connection.use { conn ->
+            conn.createStatement().use { stmt ->
+                stmt.execute("REVOKE CREATE, CONNECT, TEMPORARY, TEMP ON DATABASE ${quote(dbname)} FROM $dbUser")
```

Comment:
might be a good idea to store an array of permissions somewhere that we reference here and on granting, so that if we ever add/remove a permission we don't end up with extra floating permissions / only have to update in one place